Assets/SEE/Controls/DesktopPlayerMovement.cs

```diff
@@ -116,7 +128,8 @@ private void Update()
                 }
                 velocity.Normalize();
                 velocity *= speed;
-                transform.position += velocity;
+                controller.Move(velocity);
+                controller.Move(Vector3.zero);
```

Comment:
Similarly, a short comment on why `Move` needs to be called with the zero vector would be helpful.

Reply:
Not 100% sure why thats working, its not mentioned in the unity documentation, but it fixed the issue of a sliding avatar without input when the collider hits the floor.